frontend/src/metabase/actions/containers/ActionCreator/FormCreator/OptionEditor.tsx

```diff
@@ -16,22 +16,36 @@ import {
 } from "./OptionEditor.styled";
 
 const optionsToText = (options: FieldValueOptions) => options.join("\n");
-const textToOptions = (text: string): FieldValueOptions =>
-  text.split("\n").map(option => option.trim());
+const textToOptions = (text: string): FieldValueOptions => {
+  const options = text
+    .trim()
```

Comment:
This trim should not be necessary if we're going to trim every option afterwards

Reply:
It looks like the `Boolean` would be more important than the trim here - I played around with a test string and it looks like we can get rid of the first `trim` call as long as we filter out the empty (falsy) strings at the end
<img width="419" alt="image" src="https://github.com/metabase/metabase/assets/25306947/ff24c0bb-adf5-44f7-bb86-c0372f955837">

Reply:
trim solves the different problem - it drops all the empty spaces which would be considered as new lines and passed through the filter - it's not that optimal (I know we're not talking about perf for 5 lines, but anyway)

<img width="210" alt="image" src="https://github.com/metabase/metabase/assets/125459446/544c2f82-ac8c-4481-b61d-d3e603dbe6ac">

I'd keep trim